db/db_test.cc

```diff
@@ -5010,7 +5010,8 @@ TEST_F(DBTest, DynamicLevelCompressionPerLevel2) {
       "LevelCompactionPicker::PickCompaction:Return", [&](void* arg) {
         Compaction* compaction = reinterpret_cast<Compaction*>(arg);
         if (compaction->output_level() == 4) {
-          ASSERT_TRUE(compaction->output_compression() == kLZ4Compression);
+          ASSERT_EQ(compaction->output_compression()->GetCompressionType(),
```

Comment:
1.  I wonder if output_compression should be renamed to compressor or output_compressor?
2. Does it make sense to have a compaction->GetCompressionType shortcut method?

Reply:
Renamed output_compression to output_compressor.
I did not add the shortcut method for compression type. I see both GetCompressionType and GetId called on the compressor returned by output_compressor in different places.